src/services/evaluation-administration-service.ts

```diff
@@ -234,7 +234,84 @@ export const close = async (id: number) => {
   )
 }
 
-export const sendReminder = async (id: number) => {
+export const sendReminderByEvaluator = async (id: number, evaluator_id: number, email: string) => {
```

Comment:
I think we should pass `user_id` & `is_external` here 🤔

Reply:
Is `user_id` the same with `evaluator_id` kuys @jatnerubia ?

Reply:
Yes